### PR TITLE
Fix snmptrap alert bug

### PR DESCRIPTION
**Pull request by WingkaiHo**
Fix snmptrap alerts info omit to send the field of SnmpConstants.sysUpTime. 

So when standard snmptrapd  receive alerts package the field OID will dislocation

### CODE REVIEW

File: plugins/alert-handlers/snmp-alerts/src/org/apache/cloudstack/alert/snmp/SnmpHelper.java [hunk 1]
```diff
@@ -65,6 +66,7 @@ private PDU createPDU(SnmpTrapInfo snmpTrapInfo) {
 
         int alertType = snmpTrapInfo.getAlertType() + 1;
         if (alertType > 0) {
+            trap.add(new VariableBinding(SnmpConstants.sysUpTime, new OctetString(new Date().toString())));
```
Comment (DaanHoogland) [inline plugins/alert-handlers/snmp-alerts/src/org/apache/cloudstack/alert/snmp/SnmpHelper.java]: This doesn't seem what we want. The sysUpTime shouldn't be set to 'now', should it? Without to much diving in I'd say it should be added and retrieved from the SnmpTrapInfo. If I am wrong, please add a comment explaining why 'now' is alright.

Comment (wido) [inline plugins/alert-handlers/snmp-alerts/src/org/apache/cloudstack/alert/snmp/SnmpHelper.java]: Agree with @DaanHoogland here. sysuptime isn't indeed now(), but for how long the system has been up.

Comment (WingkaiHo) [inline plugins/alert-handlers/snmp-alerts/src/org/apache/cloudstack/alert/snmp/SnmpHelper.java]: @DaanHoogland Ok

I will use this  expression to replace now
sysUpTimeTicks = ManagementFactory.getRuntimeMXBean().getUptime() / 10;
trap.add(new VariableBinding(SnmpConstants.sysUpTime, new TimeTicks(sysUpTimeTicks)))

**Comment by DaanHoogland** [comment]
@WingkaiHo if the trap is always coming from the management server this makes sense. I am not familiar enough with this functionality.
code LGTM

**Comment by WingkaiHo** [comment]
@DaanHoogland cloudstack all alert message use the snmptrap send to third party system, if you set at log4j. And you can snmptrapd and snmptt to dispose alert. If you not add this field, the snmptrapd and snmptt do not work normal.

**Comment by cloudmonger** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 799
 Hypervisor xenserver
 NetworkType Advanced
 Passed=110
 Failed=2
 Skipped=12

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/r2si930m8xxzavs/AAAzNrnoF1fC3auFrvsKo_8-a?dl=0

**Failed tests:**
* test_volumes.py

 * test_06_download_detached_volume Failing since 2 runs

* test_routers_network_ops.py

 * test_01_RVR_Network_FW_PF_SSH_default_routes_egress_true Failing since 3 runs


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_01_verify_libvirt
test_02_verify_libvirt_after_restart
test_03_verify_libvirt_attach_disk
test_04_verify_guest_lspci
test_05_change_vm_ostype_restart
test_06_verify_guest_lspci_again
test_static_role_account_acls
test_11_ss_nfs_version_on_ssvm
test_nested_virtualization_vmware
test_3d_gpu_support
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_vm_snapshots.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_loadbalance.py
test_routers.py
test_reset_vm_on_reboot.py
test_deploy_vms_with_varied_deploymentplanners.py
test_network.py
test_router_dns.py
test_non_contigiousvlan.py
test_login.py
test_deploy_vm_iso.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_metrics_api.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_vm_life_cycle.py
test_disk_offerings.py

**Comment by rhtyd** [comment]
LGTM. Merging this based on BVT and Travis test results.

**Comment by WingkaiHo** [comment]
how to add patch to jira

**Comment by rhtyd** [comment]
@WingkaiHo go to https://issues.apache.org/jira/secure/CreateIssue!default.jspa, create an account if necessary and submit a ticket under CLOUDSTACK project. And share here the JIRA-ID (or ticket ID, something like `CLOUDSTACK-xxxxx`)

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
Previous test LGTM, the failing tests are unrelated and were known issues.
Merging this based on two lgtms and travis/trillian test results.

**Pull request merged**